CHANGELOG.md

```diff
@@ -10,13 +10,20 @@
 * Renaming symbol fails within a file that had recently been renamed without saving changes.
   * As a workaround, make an edit within the file before using Rename Symbol.
 
-## 1.23.9 (Not yet released)
+## 1.23.9 (January 29, 2021)
```

Comment:
Change the date?

Reply:
Since we aren't sure when validation and publishing will occur, we can leave the date as not released.

```suggestion
## 1.23.9 (Not yet released)
```